packages/config/src/layer3s/deri.ts

```diff
@@ -29,12 +63,208 @@ export const deri: Layer3 = {
         'https://discord.com/invite/kb8ZbYgp8M',
       ],
     },
-    dataAvailabilityMode: 'NotApplicable',
   },
   config: {
-    escrows: [],
+    escrows: [
+      discovery.getEscrowDetails({
+        // Bridge
+        address: EthereumAddress('0xD4FE46D2533E7d03382ac6cACF0547F336e59DC0'),
```

Comment:
This address has 0 ETH

Reply:
nope: 
<img width="713" alt="image" src="https://github.com/l2beat/l2beat/assets/30298476/f25a4987-61d2-4a17-af57-f563c5dbf027">